alerts.tf

```diff
@@ -26,7 +26,7 @@ module "div-server-errors-alert" {
 
   alert_name = "div-server-errors"
   alert_desc = "Found HTTP requests with 500 error response code (bad request) in div-${var.env}."
-  app_insights_query = "requests | where resultCode startswith '5'  | where name !contains '/health' | where datetime_diff('hour', timestamp, startofday(timestamp)) !between (2 .. 5) or url !contains '-aat'"
+  app_insights_query = "requests | where resultCode startswith '5' | where name !contains '/health' | where datetime_diff('hour', timestamp, startofday(timestamp)) !between (1 .. 5) or url !contains '-aat'"
```

Comment:
Is the `url !contains '-aat'` to disable logs in non-prod? In that case we should use `enabled = "${var.env == "prod"}"` (see other alerts) instead of executing the query on AAT every few mins..

Reply:
But we want server errors from AAT I thought? As per your suggestion yesterday..?

Reply:
It's to ensure we get support emails if Prod is messing up between 2-5am, but not AAT

Reply:
Sure, I missed the time bit sorry. Yes my preference would be to keep alerts (all the time fix the root cause) but given this is already in master 🤷‍♂️
Either way, I don't really know if this `datetime_diff` is working to be honest, when I run it, I get this
<img width="1440" alt="Screenshot 2019-08-09 at 09 43 09" src="https://user-images.githubusercontent.com/629600/62766415-55d84200-ba8a-11e9-9533-19630ba1cdcc.png">

And results going back to May

Reply:
These results are from PRs which don't have `-aat` in the URL (they are `-preview`) which means they'll show up. Seems to work for me. Also we don't have email alerts from after 2am UTC (search start time) in the inbox, indicating it worked but the filter needs switching to UTC as in what this PR does.

Reply:
Ah sure fair enough